### PR TITLE
Soundux: Remove obsolete dependency `pulseaudio-utils`

### PR DESCRIPTION
**Pull request by D3SOX**
We no longer require this dependency and it can safely be removed (forgot to do this in the 0.2.7 update PR, sorry)

### CODE REVIEW

File: packages/soundux/soundux.pacscript [hunk 1]
```diff
@@ -2,7 +2,7 @@ name="soundux"
 version="0.2.7"
 url="https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz"
 build_depends="build-essential cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript:4]:
```suggestion
build_depends="cmake libx11-dev libxi-dev libwebkit2gtk-4.0-dev libappindicator3-dev libssl-dev libpulse-dev"
```
`build-essential` is obsolete too, as it is a pacstall dependency, no need to add it separately

Citation
https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/install.sh#L91

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/soundux/soundux.pacscript (hunk 1 above)
Comment (D3SOX) [reply packages/soundux/soundux.pacscript:4]: Thanks!

**Review by wizard-28 — COMMENTED** [review]
Are you sure that this works without a `prepare()` function?

I tried removing the `prepare()` function from another pacscript, and I got an error.

Citation (Pacstall currently executes `prepare` regardless if it exists or not.)
https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/misc/scripts/install-local.sh#L198

**Comment by D3SOX** [comment]
> Are you sure that this works without a `prepare()` function?
> 
> I tried removing the `prepare()` function from another pacscript, and I got an error.
> 
> Error I got
> 
> ```
> /usr/share/pacstall/scripts/install-local.sh: line 199: prepare: command not found
> ```
> 
> Citation (Pacstall currently executes `prepare` regardless if it exists or not.)
> https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/misc/scripts/install-local.sh#L198

Hm I tested it a VM and the current package installed successfully. I removed the `prepare` in #14 because I thought it's unnecessary

**Comment by wizard-28** [comment]
> > Are you sure that this works without a `prepare()` function?
> > I tried removing the `prepare()` function from another pacscript, and I got an error.
> > Error I got
> > ```
> > /usr/share/pacstall/scripts/install-local.sh: line 199: prepare: command not found
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Citation (Pacstall currently executes `prepare` regardless if it exists or not.)
> > https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/misc/scripts/install-local.sh#L198
> 
> Hm I tested it a VM and the current package installed successfully. I removed the `prepare` in #14 because I thought it's unnecessary. Which version of pacstall are you using?

Latest develop branch. See the citation

**Comment by wizard-28** [comment]
> > Are you sure that this works without a `prepare()` function?
> > I tried removing the `prepare()` function from another pacscript, and I got an error.
> > Error I got
> > ```
> > /usr/share/pacstall/scripts/install-local.sh: line 199: prepare: command not found
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Citation (Pacstall currently executes `prepare` regardless if it exists or not.)
> > https://github.com/pacstall/pacstall/blob/35b3f057d4104fa6ded75e8faecc923e1ee8477d/misc/scripts/install-local.sh#L198
> 
> Hm I tested it a VM and the current package installed successfully. I removed the `prepare` in #14 because I thought it's unnecessary. Which version of pacstall are you using?

The package installed successfully but it also prints that error... Which looks bad

File: packages/soundux/soundux.pacscript [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name="soundux"
 version="0.2.7"
 url="https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz"
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript:3]:
```suggestion
url="https://github.com/Soundux/Soundux/releases/download/"$version"/soundux-"$version".tar.gz"
```
Reuse the `version` variable

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by D3SOX** [comment]
> Latest develop branch

Yeah I've used the latest release. I didn't know this is needed.

**Comment by D3SOX** [comment]
> but it also prints that error... Which looks bad

I think the prepare function should be optional. I added it for now but this should be resolved in pacstall.

File: packages/soundux/soundux.pacscript [hunk 3]
```diff
@@ -1,13 +1,17 @@
 name="soundux"
 version="0.2.7"
-url="https://github.com/Soundux/Soundux/releases/download/"$version"/soundux-"$version".tar.gz"
```
Comment (wizard-28) [inline packages/soundux/soundux.pacscript]: Why remove the `"`? It prevents word globbing and other weird things..

Comment (D3SOX) [reply packages/soundux/soundux.pacscript]: It's already in quotes

Comment (wizard-28) [reply packages/soundux/soundux.pacscript]: Hmm... ok then.

**Review by wizard-28 — APPROVED** [review]
Good to go!

**Pull request merged**